amazon-sns/4.7/modules/ROOT/pages/amazon-sns-connector-upgrade-migrate.adoc

```diff
@@ -13,7 +13,7 @@
 
 === 4.7
 
-* Added support for *FIFO (first in first out) Topic*, which together with *Amazon SQS FIFO queues* provide strict message ordering and message deduplication.
+* Added support for FIFO topics, which provide strict message ordering and message deduplication with Amazon SQS FIFO queues.
```

Comment:
I am not sure what this means. Does it mean that FIFO topics work with the queues to provide ordering and deduplication, as the original text stated? My guess is that the FIFO topics provide the queues, but that deduplication is a separate process. Since the developers are probably gone, you could leave what you have for now and look into this next week.

Reply:
I believe FIFO topics can provide those (strict message ordering and message deduplication) with Amazon SQS FIFO queues.